### ISSUE TITLE
Publish type definitions

### ISSUE THREAD

**Issue by silverwind** [un-ts/eslint-plugin-import-x#87]
In flat config format, eslint plugins are imported using `import` statements should therefor publish type definitions to avoid such typescript errors:

```
TS7016: Could not find a declaration file for module 'eslint-plugin-import-x'
```

**Comment by SunsetTechuila** [comment]
this package has types included since v0.3.0

make sure you have the latest version, which is 0.5.1

**Comment by silverwind** [comment]
I see, sorry for the noise.

https://www.npmjs.com/package/eslint-plugin-import-x?activeTab=code
